api/v1/apiserver_types.go

```diff
@@ -23,6 +23,9 @@ import (
 
 // APIServerSpec defines the desired state of Tigera API server.
 type APIServerSpec struct {
+	// Replicas defines how many Tigera API server replicas in the cluster. Defaults to 1.
```

Comment:
For typha we have an autoscaler that adjusts the replicas as needed based on cluster size, without a config knob for the user to set.

Reply:
In other words, I don't think defaulting is the way to do this. If the user sets it, we probably need to respect the user config.

If unset, we could have some default scaling behavior though

Reply:
> If unset, we could have some default scaling behavior though

I can probably implement the following for apiserver:
* if set, respect the user config.
* if unset, use 2 for 5+ nodes and 1 otherwise.

Reply:
We decided to set the default replicas to 2 independent of cluster size.